src/pipeline_manager.py

```diff
@@ -91,7 +91,24 @@ def evaluate(pipeline_name, validation_size, logger, params, ctx):
     meta = pd.read_csv(os.path.join(params.meta_dir, 'stage1_metadata.csv'))
     meta_train = meta[meta['is_train'] == 1]
     valid_ids = eval(params.valid_category_ids)
-    meta_train_split, meta_valid_split = train_valid_split(meta_train, validation_size, valid_category_ids=valid_ids)
+
+    try:
+        validation_size = float(validation_size)
+    except ValueError:
+        if validation_size=='test':
+            validation_size = 0
```

Comment:
@apyskir I don't like this artificial logic here. Maybe simple go with pass and then instead of `if validation_size > 0` go with `if isinstance(validation_size, float)`: